.github/workflows/ci.yml

```diff
@@ -6,29 +6,42 @@ on:
     branches: '*'
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
     steps:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
       - uses: wagoid/commitlint-github-action@v2
+        if: matrix.os == 'ubuntu-latest'
       - name: Set up JDK 1.8
         uses: actions/setup-java@v1
         with:
           java-version: 1.8
-      - name: Build with Maven
+      - name: Build with Maven (Windows)
         run: mvn -U -ntp clean verify
+        shell: cmd
+        continue-on-error: true
```

Comment:
Is this temporary? Is that why you're splitting this for windows and non windows?

Reply:
nvm about the splitting part, got that